identity.go

```diff
@@ -59,15 +60,22 @@ func (i Identity) Whois() string {
 }
 
 // WhoisAdmin returns a whois description for admin users.
-func (i Identity) WhoisAdmin() string {
+func (i Identity) WhoisAdmin(room *chat.Room) string {
 	ip, _, _ := net.SplitHostPort(i.RemoteAddr().String())
 	fingerprint := "(no public key)"
 	if i.PublicKey() != nil {
 		fingerprint = sshd.Fingerprint(i.PublicKey())
 	}
+
+	isOp := ""
+	if member, ok := room.MemberByID(i.ID()); ok && room.IsOp(member.User) {
+		isOp = " > op: true" + message.Newline
+	}
+
 	return "name: " + i.Name() + message.Newline +
 		" > ip: " + ip + message.Newline +
 		" > fingerprint: " + fingerprint + message.Newline +
 		" > client: " + sanitize.Data(string(i.ClientVersion()), 64) + message.Newline +
-		" > joined: " + humantime.Since(i.created) + " ago"
+		" > joined: " + humantime.Since(i.created) + " ago" + message.Newline +
```

Comment:
The whois output didn't have an extra newline at the end before, right?

I think `message.Newline` needs to go into the `isOp` prefix, otherwise you'll have an extra new line when they're not op?

Also remove the `message.Newline` from the `isOp` suffix.

Reply:
thanks for feedback, have updated it for you.